### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by konbraphat51**
## What you changed (What reviewers have to see)
Wrote `README.md`

**Look at [Top page of `documentation` branch](https://github.com/konbraphat51/AnimatedWordCloud/tree/documentation)**

## Check list
- [ ] Code styles confirmed
- [x] Test confirmed

### CODE REVIEW

**Comment by konbraphat51** [comment]
Waiting for #56

**Comment by konbraphat51** [comment]
Testing Pypi_test
https://test.pypi.org/project/WordCloudTimelapse/0.9.0/

File: README.md [hunk 1]
```diff
@@ -1,12 +1,126 @@
...
+[![unit-test](https://github.com/konbraphat51/AnimatedWordCloud/actions/workflows/python-tester.yml/badge.svg?branch=main)](https://github.com/konbraphat51/AnimatedWordCloud/actions/workflows/python-tester.yml)[![codecov](https://codecov.io/gh/konbraphat51/AnimatedWordCloud/graph/badge.svg?token=4OOX0GSJDJ)](https://codecov.io/gh/konbraphat51/AnimatedWordCloud)
+
+AnimatedWordCloudTimelapse animates the timelapse of your words vector.
+
+## Examples!
+
+Using [Elon Musk's tweets](https://data.world/adamhelsinger/elon-musk-tweets-until-4-6-17).  
+(C) Elon Musk
+
+[GIF HERE]
```
Comment (SuperHotDogCat) [inline README.md:16]: Add Gif

File: README.md [hunk 2]
```diff
@@ -1,12 +1,126 @@
...
+```
+
+### coding
+
+#### Using default configuration
+
+```python
+from AnimatedWordCloudTimelapse import animate
+
+# data must be list[("time name", dict[str, float])]
```
Comment (SuperHotDogCat) [inline README.md:39]: It is better to explain what the str and float of dict[str, float] mean.

File: setup.py [hunk 3]
```diff
@@ -4,29 +4,27 @@
...
-    version="0.1.0",
-    description="Animate a word cloud",
+    name="AnimatedWordCloudTimelapse",
+    version="0.9.0",
+    description="Animate a timelapse of word cloud",
     long_description=readme,
     long_description_content_type="text/markdown",
     author="konbraphat51, superhotdogcat",
-    author_email="",
+    author_email="konbraphat51@gmail.com",
```
Comment (SuperHotDogCat) [inline setup.py:27]: Me siromisochan@gmail.com

File: AnimatedWordCloud/Animator/Animator.py [hunk 4]
```diff
@@ -27,8 +32,27 @@ def animate(
...
+
+    # Calculate allocation
+    allocation_timelapse = allocate(timelapse_word_vector, config)
+
+    # to images
+    image_paths = create_images(allocation_timelapse, config)
+
+    # to one animation file
+    animation_path = integrate_images(image_paths, config)
+
```
Comment (SuperHotDogCat) [inline AnimatedWordCloud/Animator/Animator.py:57]: Add print(f"animation is saved in {animation_path}"), it is more user-friendly.

File: README.md [hunk 5]
```diff
@@ -1,12 +1,129 @@
...
+```
+pip install AnimatedWordCloudTimelapse
+```
+
+### coding
+
+#### Using default configuration
+
+```python
+from AnimatedWordCloudTimelapse import animate
```
Comment (SuperHotDogCat) [inline README.md:37]: Change the import path to "from AnimatedWordCloud import animate"

**Pull request merged**